platform/darwin/test/MGLDocumentationExampleTests.swift

```diff
@@ -550,10 +550,10 @@ class MGLDocumentationExampleTests: XCTestCase, MGLMapViewDelegate {
         let redColor = UIColor.red
         #endif
         let expression = NSExpression(forConstantValue: "Foo")
-        let attributes: Dictionary<MGLAttributedExpressionKey, Any> = [.fontNamesAttribute : ["DIN Offc Pro Italic",
-                                                                                              "Arial Unicode MS Regular"],
-                                                                       .fontSizeAttribute: 1.2,
-                                                                       .fontColorAttribute: redColor]
+        let attributes: Dictionary<MGLAttributedExpressionKey, NSExpression> = [.fontNamesAttribute : NSExpression(forConstantValue: ["DIN Offc Pro Italic",
```

Comment:
```suggestion
        let attributes: [MGLAttributedExpressionKey: NSExpression] = [.fontNamesAttribute : NSExpression(forConstantValue: ["DIN Offc Pro Italic",
```

(Remember to rerun `make darwin-style-code` afterwards.)